tests/contenttypes_tests/tests.py

```diff
@@ -18,7 +21,26 @@
 
 @override_settings(ROOT_URLCONF='contenttypes_tests.urls')
 class ContentTypesViewsTests(TestCase):
-    fixtures = ['testdata.json']
+
+    @classmethod
+    def setUpTestData(cls):
+        # don't use the manager because we want to update the existing instance
+        # rather than "create" a new one with pk=1
+        cls.site1 = Site(pk=1, domain='testserver', name='testserver')
+        cls.site1.save()
```

Comment:
Use `force_update` in this case to make sure an error is thrown if it doesn't exist.